src/Services/PerspectiveAccessor.php

```diff
@@ -63,6 +63,9 @@ protected function convertTreeStoreToConfiguration($treeStore)
                                         unset($grandchild['config']['treeContextMenu'][$contextMenuEntry]);
                                     }
                                 }
+                                if (empty($grandchild['config']['treeContextMenu'])) {
```

Comment:
We should avoid using the empty() function.
```suggestion
                                if ($grandchild['config']['treeContextMenu'] === []) {
```

Reply:
@blankse , what is reason for avoid using empty in this case? no see any BC regarding that in PHP 8.1, 8.2. As I see it's alot of "empty" usage in bundle code and if you have arguments to replace - it should be in scope of another PR for change all places and abit out of scope of fix.

Reply:
@blankse , in this case it's out of scope and should be changed in all places.

Reply:
@vmalyk I think it is hard to change it in all place at once. The first step would be not create new code with it...

Reply:
Thank you 👍🏼